client/components/quickview/quickview_container.jsx

```diff
@@ -44,7 +44,13 @@ QuickViewContainer.propTypes = {
   image: PropTypes.string.isRequired,
   mapSource: PropTypes.string.isRequired,
   overview: PropTypes.string.isRequired,
-  details: PropTypes.string.isRequired,
+  details: PropTypes.shape({
+    languages: PropTypes.arrayOf(PropTypes.string),
+    tour_time: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
+    vouchers_allowed: PropTypes.arrayOf(PropTypes.bool),
+    instant_confirmation: PropTypes.bool,
+    cancellations: PropTypes.bool,
```

Comment:
Is cancellations indicating if cancellations are allowed or not? The vouchers_allowed prop throws some confusion at if cancellations and other booleans are also indicating "allowed" or "not allowed" as well.

Reply:
Yeah.  I am not happy with my details data structure.  I might refactor if I have time.  I will put a ticket on it.